bcda/web/api.go

```diff
@@ -279,9 +376,6 @@ func jobStatus(w http.ResponseWriter, r *http.Request) {
 			scheme = "https"
 		}
 
-		re := regexp.MustCompile(`/(ExplanationOfBenefit|Patient|Coverage)/\$export`)
```

Comment:
Should we still have this check here, but for `/Patient` only?

Reply:
Nah we don't really need it with these changes. Its really only there to create a regex for getting the resource type from the job's request url to properly display the fhir resources in the output of the payload. That now comes from the resource type being stored in the jobKey table that gets populated during construction of the individual files themselves. So now the resource type and the filename both get stored together and retrieved together with this output.